sample-scripts/create_new_ivr/Create_New_IVR_TEMP.py

```diff
@@ -0,0 +1,167 @@
+import requests
```

Comment:
I assume that `Create_New_IVR_TEMP.py` is a copy of the old script - one where we were first generating the token and then calling the create IVR API.

This shouldn't be a part repo (this is a public repo so customers will be able to see and download all files).

I have reviewed the new file and it looks OK. Can you remove `Create_New_IVR_TEMP.py` from the pull request? We can merge the request then.

Reply:
`Create_New_IVR_TEMP.py` Deleted